examples/undocumented/libshogun/splitting_LOO_crossvalidation.cpp

```diff
@@ -38,14 +38,12 @@ int main(int argc, char **argv)
 		CLOOCrossValidationSplitting* splitting=
 				new CLOOCrossValidationSplitting(labels);
 
-		splitting->build_subsets();
-
 		for (index_t i=0; i<num_labels; ++i)
 		{
 			//SG_SPRINT("subset %d\n", i);
 
-			SGVector<index_t> subset=splitting->generate_subset_indices(i);
-			SGVector<index_t> inverse=splitting->generate_subset_inverse(i);
+			SGVector<index_t> subset = splitting->validation(i);
+			SGVector<index_t> inverse = splitting->train(i);
```

Comment:
just realised a minor inconsistency. Should be called "training" if the counterpart is called "validation"
But I would even prefer "train" and "test" ..